homeassistant/components/spotify/strings.json

```diff
@@ -16,6 +16,9 @@
     "create_entry": {
       "default": "Successfully authenticated with Spotify."
     },
+    "initiate_flow": {
```

Comment:
initiate_flow is too generic, we also have options flow.

Reply:
Yes, so for the subentries we have `initiate_flow`, this just reuses it
https://github.com/home-assistant/core/blob/dev/homeassistant/components/google_weather/strings.json#L48-L50